src/magellan_core/msg/Float64Arr.msg

```diff
@@ -0,0 +1 @@
+float64[] data
```

Comment:
So the ROS message naming convention would mean that this message should be named `Float64Array`. Additionally, we should be adding a header to signify what the frame is and when the lines were detected. I would follow what Levi did here https://github.com/Pitt-RAS/iarc7_msgs/blob/master/msg/Float64ArrayStamped.msg